setup.py

```diff
@@ -31,6 +31,8 @@
         'requests<3.0.0',
         'beautifulsoup4',
         'posthog',
+        'boto3',
+        'google-cloud-storage',
```

Comment:
mmm maybe it will be better to make them as extras requirements?
I think that installing boto3/gcs when there is no use of it can be annoying.
WDYT?

Reply:
I don't want our package to be _too_ modular in the sense that our users would need to think too much about which features to install and how.

I looked into the packages' requirements and they look very lean and only relevant to their domain so the chances of conflicts are quite low.

`google-cloud-storage`
Requires: google-api-core, google-auth, google-cloud-core, google-resumable-media, requests

`boto3`
Requires: botocore, jmespath, s3transfer